eng/pipelines/templates/steps/test-conda.yml

```diff
@@ -0,0 +1,95 @@
+parameters:
+  - name: TestPipeline
+    type: boolean
+    default: false
+  - name: ServiceDirectory
+    type: string
+    default: ''
+  - name: CondaArtifacts
+    type: object
+    default: []
+  - name: TestMarkArgument
+    type: string
+    default: ''
+  - name: PythonVersion
+    type: string
+    default: ''
+  - name: OSVmImage
+    type: string
+    default: ''
+
+steps:
+  - task: DownloadPipelineArtifact@2
+    inputs:
+      artifactName: 'conda'
+      targetPath: $(Build.ArtifactStagingDirectory)
+
+  - template: /eng/common/pipelines/templates/steps/set-test-pipeline-version.yml
+    parameters:
+      PackageName: "azure-template"
+      ServiceDirectory: "template"
+      TestPipeline: ${{ parameters.TestPipeline }}
+
+  - task: UsePythonVersion@0
+    displayName: 'Use Python $(PythonVersion)'
+    inputs:
+      versionSpec: $(PythonVersion)
+
+  - ${{ each artifact in parameters.CondaArtifacts }}:
+    # there may be multiple CondaArtifacts. Be certain $(conda.build) is clean just in case!
+    - pwsh:
+        Write-Host "Clean up Conda Build Directory $(conda.build)"
+        Remove-Item $(conda.build)/* -Recurse -Force
+      displayName: 'Clean Up Before Building ${{ artifact.name }}'
+
+    - ${{ each checkout in artifact.checkout }}:
+      - template: /eng/pipelines/templates/steps/get-tagged-code.yml
+        parameters:
+          DestinationDirectory: $(conda.build)/${{checkout.package}}
+          Package: ${{checkout.package}}
+          CheckoutPath: ${{checkout.checkout_path}}
+          Version: ${{checkout.version}}
+
+    - task: PythonScript@0
+      displayName: 'Build Source Distribution for ${{ artifact.name }}'
+      inputs:
+        scriptPath: 'scripts/devops_tasks/build_conda_artifacts.py'
+        arguments: >-
+          -d "$(conda.output)"
+          -b "$(conda.build)"
+          -m "$(Build.SourcesDirectory)/sdk/${{ parameters.ServiceDirectory }}/${{ artifact.meta_source }}"
+          -r "${{ artifact.common_root }}"
+          -n "${{ artifact.name }}"
+          -s "${{ parameters.ServiceDirectory }}"
+          -e "$(Build.SourcesDirectory)/eng/conda_env.yml"
+          -c "$(Build.SourcesDirectory)/sdk/${{ parameters.ServiceDirectory }}/ci.yml"
+
+    - bash: |
```

Comment:
Is thee a reason you are using bash here in this template? It will end up limiting its usage to only linux. Do we need to ever run these tests on other platforms?

Reply:
Totally ack. It's going to have to change. I'm just trying to get to the place where it's running -> get it working cross plat.